include/ql/ir/prim.h

```diff
@@ -331,6 +331,7 @@ std::ostream &operator<<(std::ostream &os, const Matrix<T> &mat) {
     return os;
 }
 
+#undef IGNORE
```

Comment:
There is an IGNORE macro defined in a Windows header winbase.h 😮 
Windows 😭

Reply:
I would prefer to redefine our IGNORE
Now all users of this header file must check that they don't use a facility that on windows might depend on the setting of this IGNORE macro

Reply:
Good point, will do, in same PR but in a preliminary commit not to mess things up.